x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/suggestion_panel.scss

```diff
@@ -1,21 +1,57 @@
-.lnsSidebar__suggestions {
-  > * {
-    margin-top: $euiSizeS;
+ // SASSTODO: Create this in EUI
```

Comment:
This is nice. Made an issue so we remember.

https://github.com/elastic/eui/issues/2215

Reply:
TY!